src/nni_manager/training_service/remote_machine/extends/linuxCommands.ts

```diff
@@ -22,10 +22,10 @@ class LinuxCommands extends OsCommands {
             export NNI_PLATFORM=remote NNI_SYS_DIR=${workingDirectory} NNI_OUTPUT_DIR=${workingDirectory} NNI_TRIAL_JOB_ID=${trialJobId} \
             NNI_EXP_ID=${experimentId} NNI_TRIAL_SEQ_ID=${trialSequenceId} NNI_CODE_DIR=${codeDir}
             export MULTI_PHASE=${isMultiPhase}
-
-            cp -r $NNI_CODE_DIR/. $NNI_SYS_DIR
-            cd $NNI_SYS_DIR
-            sh install_nni.sh
+            mkdir -p $NNI_SYS_DIR/code
```

Comment:
It will break trial status check with extra `code` folder. As nnictl need check `.nni` folder, and it may break.
If it's important to keep code folder clean, the folders need to be changed throughtly.

Reply:
The .nni folder is stroed in `NNI_SYS_DIR`, users' code is tored in `$NNI_SYS_DIR/code`, they are two paralled path, NNI SDK will write metics to `$NNI_SYS_DIR/.nni` folder, the path of code files does not effect this logic.